packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/statistics/statistics.controller.js

```diff
@@ -538,4 +546,100 @@ export default class XdslStatisticsCtrl {
         this.attenuation.loading = false;
       });
   }
+
+  /**
+   * Get errors statistic for the line
+   * @param {String} period Period to request :
+   *          - daily
+   *          - monthly
+   *          - preview
+   *          - weekly
+   *          - yearly
+   * @return {promise}
+   */
+  getErrorsStatistics(period) {
+    this.errors.loading = true;
+    return this.$q
+      .all({
+        hec: this.getLinesStatistics('error:hec', period),
+        fec: this.getLinesStatistics('error:fec', period),
+        crc: this.getLinesStatistics('error:crc', period),
+      })
+      .then((stats) => {
+        this.errors.haveSeries = !!(
+          stats.hec.length &&
+          stats.fec.length &&
+          stats.crc.length
```

Comment:
Maybe we can do something like 
```suggestion
          Object.values(stats).every(stat => stat.length)
```
so we don't have to update everytime a stat is added